lib/RAST_SDK/AnnotationUtils.pm

```diff
@@ -2313,12 +2312,12 @@ sub _fetch_object_info {
 
 # create a 12 char string unique enough here
 sub _create_uuid {
-    my $self = shift;
-
-    my $str_uuid = qx(uuidgen);
-    chomp $str_uuid;
-    $str_uuid = substr($str_uuid, -12);
-    return $str_uuid;
+    return Data::UUID->new()->create_str();
```

Comment:
use Data::UUID to generate unique IDs

Reply:
Are we sure this new code does the same thing as the old code? Should there be a substr in there?

Reply:
The method is used to generate a random, unique UUID. The existing code used a command-line program, so the results need some tidying, whereas Data::UUID doesn't require that.